Qausim/react-weather-app
# Implement Main Weather Details Component

#### What does this PR do?
Implements the component that displays the central weather details

#### Description of Task to be completed?
Implement a static component for the main weather details

#### How should this be manually tested?
Clone this repo, cd into it and 
- Run `npm start`, then visit `localhost:8080`, there should a static weather details component

#### Any background context you want to provide?
There should a main weather view

#### Questions:
None
